content/w/axel-fox.md

```diff
@@ -15,52 +15,83 @@ source = "Facebook @OficjalnePPW"
+
+#### Championships pursuits
+
+In early 2024, Axel Fox was involved as one of the contenders at the [Gold Rush](@/e/ptw/2024-02-03-ptw-5-gold-rush.md) Royal Rumble match, where he was one of the final two competitors with [Puncher](@/w/puncher.md), the final winner. Later on he was trying to win [PTW Tag Team Championship](@/c/ptw-tag-team-championship.md) from Budapest Bastards while teaming with [Marcelito](@/w/marcelito.md). It was unsucessful, same as his title fight against Marcelito and Scotty Rawk for British Wrestling Revolution Championship. After the end of the fight Axel teased a possible heelturn or at least rivarly with Marcelito, but due to his departure from PTW it led to nothing.
+
+Axel was one of the few wrestlers who kept his [PTW exit](@/a/ptw-exits.md) to himself. His departure was however confirmed by [Arkadiusz Pawłowski](@/w/pan-pawlowski.md) on one of his livestreams.
 
 During his tenure in PTW, Axel become one of the most active Polish pro wrestlers on the European indy scene, making regular appearances in Germany for wXw Academy, wXw itself and That's Awesome Wrestling; in Austria for Rings of Europe and the PTW-associated WWA; and in Hungary for HCW and Passion Pro Wrestling.
 
-He also became a part-time coach, training others at the PTW Performance Center.
+### PpW Ewenement (2024-present)
 
-### 2024: PpW
+#### GobLis and facing Zmowa
 
 During [Ale Grzeje](@/e/ppw/2024-07-13-ppw-ale-grzeje.md) in July 2024, Fox made his PpW debut as the surprise tag team partner of [Goblin](@/w/goblin.md), who was booked for a match against Legia Łysych ([Olgierd](@/w/olgierd.md) and [Marco Hammers](@/w/marco-hammers.md)).
 Fans quickly named this new tag team of Goblin and Axel "Goblis" (a portmanteau of "Goblin" and "Lis").
-They were however defeated, after an intervention by [Mister Z](@/w/mister-z.md).
+They were however defeated, after an intervention by [Mister Z](@/w/mister-z.md). Axel quickly joined "Zmowa Przeciwko Zmowie" led by [Biesiad](@/w/biesiad.md) in their conflict with ["Zmowa"](@/a/the-collusion.md). Axel developed personal rivarly against Mister Z, who frequently avoided confrontation, and viciously attacked him with a kendo stick. At [Co za Noc](@/e/ppw/2024-10-26-ppw-co-za-noc.md) Zmowa was defeated in Ustawka Match, but Axel continued his rivarly with PpW's producer.
+
+Shortly after Ale Grzeje, PpW announced that Axel Fox would become a permanent coach in [Ewenement Dojo](@/o/ewenement-dojo.md), and that he's now a full-time member of their roster.
 
-Shortly after the show, PpW announced that Axel Fox would become a permanent coach in [Ewenement Dojo](@/o/ewenement-dojo.md), and that he's now a full-time member of their roster.
+## In wrestling
+
+* Finishers:
+  - _Firefox DDT_ (2021-)
+  - _Lisia Bombka_ (Diving Elbow Drop) (2023-)
+ 
+* Tag teams and stables:
+  - FoxJoy (2023)
+  - GobLis (2024-)
+
+* Entrance music:
```

Comment:
Zakres dat za badge'ami, po datach <br> dla wymuszenia przełamania linii - bez tego przy dłuższych wpisach źle to wygląda.

(Linki na spokojnie poszukam później.)